### ISSUE TITLE
System menu is not accessible in New ms team app with NVDA

### ISSUE THREAD

**Issue by gauravahir28** [nvaccess/nvda#16177]
### Steps to reproduce:
Open the new MS teams app and press "alt + space" to bring the system menu.
### Actual behavior:

NVDA stays blank.
### Expected behavior:

NVDA should read "System menu" when it appear, and allow user to navigate using arrow key within it.
#### NVDA installed/portable/running from source:
Installed
#### NVDA version:
NVDA 2023.3.3
#### Windows version:
Windows 11 23H2
#### Name and version of other software in use when reproducing the issue:
New Microsoft Teams version 24004.1307.2669.7070.
#### Does the issue still occur after restarting your computer?
Yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
Not tried.
#### If NVDA add-ons are disabled, is your problem still occurring?
Yes
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Yes

**Comment by seanbudd** [comment]
This needs to be reported to Microsoft. Testing across multiple versions of teams showed that this dialog was not designed to be accessible.

**Comment by gauravahir28** [comment]
@seanbudd  - but this menu seems to be accessible with Narrator!

**Comment by Adriani90** [comment]
It works as expected in Windows 11 24 H2, NVDA 2025.1 Beta and Teams 25094.310.3616.953.
Closing as works for me. if you are still having this issue, please comment and we can reopen.
